core/src/main/scala/tamer/config.scala

```diff
@@ -53,6 +69,8 @@ object KafkaConfig {
     bufferSize = bufferSize,
     sink = sink,
     state = state,
+    groupId = groupId,
+    clientId = clientId,
```

Comment:
these aren't inherently properties of the state, but more of a consumer, lifting here. breaking change as it impacts the way the config is looked for (hence the env variable)